test/e2e/autotls/domain_mapping_test.go

```diff
@@ -0,0 +1,139 @@
+// +build e2e
+
+/*
+Copyright 2021 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package autotls
+
+import (
+	"context"
+	"testing"
+
+	"github.com/kelseyhightower/envconfig"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"knative.dev/networking/test/conformance/ingress"
+	duckv1 "knative.dev/pkg/apis/duck/v1"
+	"knative.dev/pkg/reconciler"
+	v1 "knative.dev/serving/pkg/apis/serving/v1"
+	"knative.dev/serving/pkg/apis/serving/v1alpha1"
+	"knative.dev/serving/test"
+	"knative.dev/serving/test/e2e"
+	v1test "knative.dev/serving/test/v1"
+)
+
+type dmConfig struct {
+	// TLSServiceName is the name of testing Knative Service.
+	// It is not required for self-signed CA or for the HTTP01 challenge when wildcard domain
+	// is mapped to the Ingress IP.
+	TLSServiceName string `envconfig:"tls_service_name" required:"false"`
+	// TLSTestNamespace is the namespace of where the tls tests run.
+	TLSTestNamespace string `envconfig:"tls_test_namespace" required:"false"`
+	// CustomDomainSuffix is the custom domain used for the domainMapping.
+	CustomDomainSuffix string `envconfig:"custom_domain_suffix" required:"false"`
+}
+
+func TestDomainMappingAutoTLS(t *testing.T) {
+	t.Parallel()
```

Comment:
```suggestion
        if !test.ServingFlags.EnableAlphaFeatures {
                t.Skip("Alpha features not enabled")
        }
	t.Parallel()
```

DomainMapping is still alpha so we probably should check the flag like other domainmapping test https://github.com/knative/serving/blob/568f0ca837f7e26f54f81421eb4135304793d8a9/test/conformance/api/v1alpha1/domain_mapping_test.go#L38

Reply:
@YoussB sorry could you please fix the format error? I think it is an indent issue.